Sources/SwiftProtobuf/AnyMessageStorage.swift

```diff
@@ -51,7 +51,11 @@ fileprivate func emitVerboseTextForm(visitor: inout TextFormatEncodingVisitor, m
 fileprivate func asJSONObject(body: Data) -> Data {
   let asciiOpenCurlyBracket = UInt8(ascii: "{")
   let asciiCloseCurlyBracket = UInt8(ascii: "}")
+  #if swift(>=5.0)
```

Comment:
Is the conditional really needed here?

I just tried the new version `Data([asciiOpenCurlyBracket])` in Swift 4.0 and it seems to work correctly there.